src/components/form/TextField.tsx

```diff
@@ -0,0 +1,26 @@
+import { FormControl, FormField, FormItem, FormLabel, FormMessage } from "../ui/form"
+import { Input } from "../ui/input"
+type Props = {
+    formControl: any
+    name: string
+    label: string
+    placeholder?: string
+    type?: "text" | "password"
```

Comment:
We should follow the type definition used by the ShadCn's `Input` component (`React.HTMLInputTypeAttribute`)

```suggestion
    type?: React.HTMLInputTypeAttribute
```